### ISSUE TITLE
labelling x and y axis of the wtr.heat.map - no error detected but nothing changes on my heat map

### ISSUE THREAD

**Issue by heatheranneshaw-13** [GLEON/rLakeAnalyzer#122]
I'm using the following code to create a heat map with rlakeanalyzer. I'm having problems when I add functions. 

When I run 

TalusSouthHeatMap <- wtr.heat.map(TalusSouthdf) 

I get a graph with a labelled y axis (Depth m). When I add the zlim function the y axis label disappears, and when I try to add the ylab, xlab and plot title nothing happens. No error is detected but nothing on my plot view changes.  

This is the code I'm using when all the functions are added. 

TalusSouthHeatMap <- wtr.heat.map(TalusSouthdf, zlim=c(0,20), xlab='Date', ylab="Depth (m)", plot.title="Talus South Water Temp (C)")

**Comment by heatheranneshaw-13** [comment]
image of my heat map for reference

![TalusNorthHeatMap](https://github.com/GLEON/rLakeAnalyzer/assets/156720745/5429d42b-9696-4514-ac8a-3640ff13c5e4)

**Comment by spencer-tassone** [comment]
Rather than using `plot.title = ` use `main = `.
``` r
library(rLakeAnalyzer)

wtr = load.ts(system.file('extdata',
                          'sparkling.wtr', package="LakeMetabolizer"))

wtr.heat.map(wtr, main = 'Sparkling Lake Water Temp (C)')
```

![](https://i.imgur.com/mcZz4Im.png)

``` r
wtr.heat.map(wtr, zlim = c(0,15), main = 'Sparkling Lake Water Temp (C)')
```

![](https://i.imgur.com/XG3rrBb.png)

<sup>Created on 2024-12-12 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>
